### PR TITLE
Document new login intent TTL

### PR DESCRIPTION
**Pull request by spwitt**
### Issue
- https://github.com/FusionAuth/fusionauth-issues/issues/2736
- https://linear.app/fusionauth/issue/ENG-988/[bug]-cannot-log-in-anymore-after-upgrade-to-v150x

### Changes
Add new external identifier duration to tenant configuration.

### Depends on
- https://github.com/FusionAuth/fusionauth-app/pull/452

### CODE REVIEW

File: .github/knownwords.txt
Comment (mooreds) [inline .github/knownwords.txt:1103]: Would still like one of the other suggestions I made on the PR instead of 'completionToken'.

Comment (spwitt) [reply .github/knownwords.txt:1103]: Understood. I'm still waiting on another review for that one to get more feedback on the naming to avoid multiple changes. I decided to get PRs for the other repos prepped ahead of time. Then I can just run through for naming instead of delaying all the work until the app PR is reviewed.

**Review by lyleschemmerling — APPROVED** [review]
no notes

**Pull request merged**
